### ISSUE TITLE
Pscanrule Content Type Missing not working.

### ISSUE THREAD

**Issue by egerod** [zaproxy/zaproxy#3849]
Hello,

Found that when scanning a site containing pages missing the "Content-Type" header, the "Content Type Missing" alert described [here](https://github.com/zaproxy/zap-core-help/wiki/HelpAddonsPscanrulesPscanrules#content-type-missing ), isn't created.

Looking at the log (debug logging enabled) I get the following, for the pages missing the Content-Type header.
```
DEBUG ParseFilter - Resource is not text: <pageUrl>
DEBUG SpiderTask - Resource fetched, but will not be parsed due to a ParseFilter rule: <pageUrl>
```
It appears that because the ParseFilter blocks the response, it is never processed, and the alert is therefor never created.

**Steps to reproduce:** Scan a page without a Content-Type header.
**Expected Outcome:** Content Type Missing Alert
**Outcome:** No alert

For a workaround to the issue see https://groups.google.com/d/msg/zaproxy-users/fjacZJR3LXo/SAJhoQRYAQAJ

**Comment by egerod** [comment]
It appears this have been fixed in the dev version. Closing the issue.

**Comment by thc202** [comment]
This was fixed with the changes done for #3392.
